mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc

```diff
@@ -0,0 +1,25 @@
+##
```

Comment:
Good ideas, probably I should re-do this to handle all common variants of on, ON, On, off, OFF, Off, true, TRUE, True, false, FALSE, and False and send it upstream.